members/I000055.yaml

```diff
@@ -53,6 +53,12 @@ contact_form:
         required: true
         options:
           max_length: 10000
+    - javascript:
+        - name: Message
+          selectors: [ "textarea#field_4BBB4940-5FA9-43E1-9F05-0A9EB169C378" ]
+          values: [ "$MESSAGE" ]
+          commands: [ "elements[0].value = values[0].replace(/www\\.youtube\\.com\\/watch?[^&]*(&)?v=/gi, 'youtu.be/')" ]
+          required: true
```

Comment:
did this replace the $MESSAGE above? Also need the max_length

Reply:
I have a question about this. By adding two $MESSAGE variables, does that mean that one replaces the other? In this case, would the JS step replace the previous $MESSAGE step? I assumed both variables were kept and used when they are present, but I guess that's not the case. Not sure how to handle this. 

For instance, I did a test adding an option to the JS step, like so
    - javascript:
        - name: Message
          selectors: [ "#message" ]
          values: [ "$MESSAGE" ]
          commands: [ "elements[0].value = values[0].replace(/www\\.youtube\\.com\\/watch?[^&]*(&)?v=/gi, 'youtu.be/')" ]
          required: true
          options:
              blacklist: "*" 

but saw that "*" was then added as a type of message option instead of just blacklisting it.  Hence why I kept both variables. 

Any suggestions?